mappings/net/minecraft/client/render/GameRenderer.mapping

```diff
@@ -46,14 +47,15 @@ CLASS net/minecraft/class_757 net/minecraft/client/render/GameRenderer
 	METHOD method_3188 renderWorld (FJLnet/minecraft/class_4587;)V
 		ARG 1 tickDelta
 		ARG 2 limitTime
+		ARG 4 matrix
 	METHOD method_3189 showFloatingItem (Lnet/minecraft/class_1799;)V
 		ARG 1 floatingItem
 	METHOD method_3190 updateTargetedEntity (F)V
 		ARG 1 tickDelta
 	METHOD method_3192 render (FJZ)V
 		ARG 1 tickDelta
 		ARG 2 startTime
-		ARG 4 fullRender
+		ARG 4 tick
```

Comment:
From here, it seems to be `inTick` (render as a part of a tick)? Or if this render method will increase the tick count (`ticking`)?
```suggestion
		ARG 4 inTick
```

Reply:
This does exactly the same thing as the one in `MinecraftClient.render`. I'll update it along with the other one if we find a better name.